cms/server/admin/handlers/subtask_validators.py

```
@@ -0,0 +1,919 @@
+    exe_name = "validator"
+    if language is not None and isinstance(language, CompiledLanguage):
+        exe_name += language.executable_extension
+
+    # Build command once outside the loop - same for all testcases
+    cmd = ["./" + exe_name, "input.txt", "output.txt"]
+    if language is not None:
+        try:
+            cmds = language.get_evaluation_commands(exe_name)
+            if cmds:
+                cmd = cmds[0] + ["input.txt", "output.txt"]
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)